app/services/weets.js

```diff
@@ -0,0 +1,17 @@
+const axios = require('axios').default;
+
+const logger = require('../logger');
+const config = require('../../config').common.apiGeekJoke;
+
+exports.getRandomJoke = async () => {
+  try {
+    const response = await axios({
+      url: `${config.url}/api?format=json`,
```

Comment:
I think that config to refer to the url does not look very good, something like config.common.apiGeekJoke would be better